bean.go

```diff
@@ -391,6 +397,11 @@ func (b *Bean) InitDB() {
 	}
 }
 
+// The bean Logger to have debug log from anywhere.
+func Logger() echo.Logger {
+	return beanLogger
```

Comment:
I think better to init the logger if it is nil
```suggestion
        if beanLogger == nil {
                beanLogger = log.New(...)
                beanLogger.SetLevel(...)
                ...
        }
	return beanLogger
```

Reply:
@yu-yk Thanks for your feedback. However, I already initialize `beanLogger` at line 187. So the expectation is that start.go should call `NewEcho()` to bootup the framework which will initialize the `beanLogger`. 

So, without booting up the framework if someone call bean.Logger() then it's clearly a wrong way to do things. That's why I didn't check `beanLogger == nil` or not from Logger().